lib/fetchAll.js

```diff
@@ -3,18 +3,19 @@ const newsAPIKey = process.env.NEWS_API     // A file stored locally (or in Hero
 const newsapi = new NewsAPI(newsAPIKey);
 
 // Calls the NewsAPI and gets the articles
-function fetchAll(store, callback){
-    store.remove('all', function(err) {     // Empty the file
+function fetchAll(store, category, callback){
+    store.remove(category, function(err) {     // Empty the file
         // If (err) throw err // err if the file removal failed <- commented out because it didn't exist
     })
 
     newsapi.v2.topHeadlines({
         language: 'en',
-        country: 'us'
+        country: 'us',
+        category: ((category=='all')? undefined: category)
```

Comment:
When possible, in javascript use `===` over `==`. `==` is less strict and less predictable than `===` (same with `!=` vs `!==`)